### PR TITLE
ssi app updates to enable use of usbserial for DCL connectivity

### PR DESCRIPTION
**Pull request by murthyvedula**
ssi comms updated to enable use of usb-serial for DCL connectivity, also improves connectivity issues observed with some USB-UART adapter cables

### CODE REVIEW

**Review by anthonyle-ql — APPROVED** [review]

**Comment by cdknorow** [comment]
@murthyvedula what information are you using from the model_json.h? This could end up being a very large string.

**Comment by murthyvedula** [comment]
> 
> 
> @murthyvedula what information are you using from the model_json.h? This could end up being a very large string.

reverted changes to model_json.h and model_json.c, information in model_json.h goes to the QLSM header, minimal info needed when parsing these QLSM files offline could go in this model_json.h file.

File: qf_apps/qf_ssi_ai_app/inc/Fw_global_config.h [hunk 1]
```diff
@@ -88,8 +99,8 @@
 #define SENSOR_COMMS_KNOWN_PATTERN (0) // 1 => Send a known sawtooth pattern for live-streaming
 
 /* Settings for selecting either Audio or an I2C sensor, Enable only one of these mode */
-#define SSI_SENSOR_SELECT_AUDIO    (0) // 1 => Select Audio data for live-streaming or recognition modes
-#define SSI_SENSOR_SELECT_SSSS     (1) // 1 => Select SSSS sensor data for live-streaming of recognition modes
+#define SSI_SENSOR_SELECT_AUDIO    (1) // 1 => Select Audio data for live-streaming or recognition modes
```
Comment (cdknorow) [inline qf_apps/qf_ssi_ai_app/inc/Fw_global_config.h:102]: accidently switched this

**Review by cdknorow — APPROVED** [review]

File: qf_apps/qf_ssi_ai_app/inc/Fw_global_config.h (hunk 1 above)
Comment (murthyvedula) [reply qf_apps/qf_ssi_ai_app/inc/Fw_global_config.h:102]: @cdknorow , default setting is I2C sensor selection

**Pull request merged**